components/locale/fr_FR.ts

```diff
@@ -33,16 +33,16 @@ const localeValues: Locale = {
     triggerAsc: 'Trier par ordre croissant',
     cancelSort: 'Annuler le tri',
   },
-  Modal: {
-    okText: 'OK',
-    cancelText: 'Annuler',
-    justOkText: 'OK',
-  },
   Tour: {
     Next: 'Étape suivante',
     Previous: 'Étape précédente',
     Finish: 'Fin de la visite guidée',
   },
+  Modal: {
+    okText: 'OK',
+    cancelText: 'Annuler',
+    justOkText: 'OK',
+  },
   Popconfirm: {
```

Comment:
Why is there this change record? Can you help remove it?

Reply:
In the other files tour is also before Modal so I thought this would make it easier to compare in the future